forms-web-app/src/middleware/fetch-existing-appeal.js

```diff
@@ -1,5 +1,3 @@
-const { getExistingAppeal } = require('../lib/appeals-api-wrapper');
-
 /**
```

Comment:
Not related to this file but above - `forms-web-app/src/middleware/.Rhistory` - what's this file?